CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Add new brand colour for FCDO
```

Comment:
Can this be moved under the 'New features' heading, and include a H4 and a link to the pull request?

e.g.

```
#### Add new brand colour for The Foreign, Commonwealth and Development Office (FCDO)

This was added in [pull request #1918: Add new brand colour for FCDO](https://github.com/alphagov/govuk-frontend/pull/1918).
```

Reply:
Thanks @36degrees ! I've updated the changelog file now 👍